packages/x/xsound.json

```diff
@@ -23,7 +25,118 @@
+          "types/SoundModule/Effectors/Effector.d.ts",
+          "types/SoundModule/Effectors/Effector.d.ts.map",
+          "types/SoundModule/Effectors/EnvelopeGenerator.d.ts",
+          "types/SoundModule/Effectors/EnvelopeGenerator.d.ts.map",
+          "types/SoundModule/Effectors/Equalizer.d.ts",
+          "types/SoundModule/Effectors/Equalizer.d.ts.map",
+          "types/SoundModule/Effectors/Filter.d.ts",
+          "types/SoundModule/Effectors/Filter.d.ts.map",
+          "types/SoundModule/Effectors/Flanger.d.ts",
+          "types/SoundModule/Effectors/Flanger.d.ts.map",
+          "types/SoundModule/Effectors/Listener.d.ts",
+          "types/SoundModule/Effectors/Listener.d.ts.map",
+          "types/SoundModule/Effectors/Panner.d.ts",
+          "types/SoundModule/Effectors/Panner.d.ts.map",
+          "types/SoundModule/Effectors/Phaser.d.ts",
+          "types/SoundModule/Effectors/Phaser.d.ts.map",
+          "types/SoundModule/Effectors/PitchShifter.d.ts",
+          "types/SoundModule/Effectors/PitchShifter.d.ts.map",
+          "types/SoundModule/Effectors/Reverb.d.ts",
+          "types/SoundModule/Effectors/Reverb.d.ts.map",
+          "types/SoundModule/Effectors/Ringmodulator.d.ts",
+          "types/SoundModule/Effectors/Ringmodulator.d.ts.map",
+          "types/SoundModule/Effectors/Stereo.d.ts",
+          "types/SoundModule/Effectors/Stereo.d.ts.map",
+          "types/SoundModule/Effectors/Tremolo.d.ts",
+          "types/SoundModule/Effectors/Tremolo.d.ts.map",
+          "types/SoundModule/Effectors/VocalCanceler.d.ts",
+          "types/SoundModule/Effectors/VocalCanceler.d.ts.map",
+          "types/SoundModule/Effectors/Wah.d.ts",
+          "types/SoundModule/Effectors/Wah.d.ts.map",
+          "types/SoundModule/Recorder/Channel.d.ts",
+          "types/SoundModule/Recorder/Channel.d.ts.map",
+          "types/SoundModule/Recorder/Track.d.ts",
+          "types/SoundModule/Recorder/Track.d.ts.map",
+          "types/SoundModule/Recorder/index.d.ts",
+          "types/SoundModule/Recorder/index.d.ts.map",
+          "types/SoundModule/Session/Room.d.ts",
+          "types/SoundModule/Session/Room.d.ts.map",
+          "types/SoundModule/Session/index.d.ts",
+          "types/SoundModule/Session/index.d.ts.map",
+          "types/SoundModule/index.d.ts",
+          "types/SoundModule/index.d.ts.map",
+          "types/StreamModule/NoiseGate.d.ts",
+          "types/StreamModule/NoiseGate.d.ts.map",
+          "types/StreamModule/NoiseSuppressor.d.ts",
+          "types/StreamModule/NoiseSuppressor.d.ts.map",
+          "types/StreamModule/index.d.ts",
+          "types/StreamModule/index.d.ts.map",
+          "types/XSound/index.d.ts",
+          "types/XSound/index.d.ts.map",
+          "types/interfaces.d.ts",
+          "types/interfaces.d.ts.map",
+          "types/main.d.ts",
+          "types/main.d.ts.map",
+          "types/types.d.ts",
+          "types/types.d.ts.map",
```

Comment:
Instead of listing all these files individually, can we make use of a glob here, please?

`types/**/*.d.ts` would probably be safe enough, or you can be more specific if you'd like to be.

Reply:
Thanks.
Fix to use a glob instead of paths each file on https://github.com/cdnjs/packages/pull/1304/files#diff-8ce6a164c124f3a2ca7ea8446f6fd74eaf30ded7c6630c3b9289d4270525972aR28-R29.